externs/browser/w3c_indexeddb.js

```diff
@@ -386,15 +445,10 @@ IDBDatabase.prototype.createObjectStore =
 IDBDatabase.prototype.deleteObjectStore = function(name) {};
 
 /**
- * @param {string} version The new version of the database.
- * @return {!IDBRequest} The IDBRequest object.
- */
-IDBDatabase.prototype.setVersion = function(version) {};
-
-/**
- * @param {string|Array<string>} storeNames The stores to open in this
- *     transaction.
- * @param {(number|string)=} mode The mode for opening the object stores.
+ * TODO: Change mode param to !IDBTransactionMode once CC understands string literal types.
+ * 
+ * @param {(string|!Array<string>)} storeNames The stores to open in this transaction.
+ * @param {(!IDBTransactionMode|string)=} mode The mode for opening the object stores.
```

Comment:
`number` here was last specced in [Working Draft 2011](https://www.w3.org/TR/2011/WD-IndexedDB-20111206/) (6 versions before final recomendation). I don't believe we can meet a browser requiring `number` as a second param.